modules/drivers/lidar/zvision/driver/zvision_driver_component.h

```diff
@@ -0,0 +1,60 @@
+
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+#pragma once
+
+#include <memory>
+#include <string>
+#include <thread>
+
+#include "modules/drivers/lidar/proto/zvision.pb.h"
+#include "modules/drivers/lidar/proto/zvision_config.pb.h"
+
+#include "cyber/cyber.h"
+#include "modules/drivers/lidar/zvision/driver/zvision_driver.h"
+
+namespace apollo
+{
+  namespace drivers
+  {
+    namespace zvision
+    {
+
+      using apollo::cyber::Component;
+      using apollo::cyber::Reader;
+      using apollo::cyber::Writer;
+      using apollo::drivers::zvision::ZvisionScan;
+
+      class ZvisionDriverComponent : public Component<>
```

Comment:
we recommend write lidar driver use common framework to get a well performance and standardization. see https://apollo.baidu.com/community/Apollo-Homepage-Document?doc=BYFxAcGcC4HpYIbgPYBtXIHQCMEEsATAV0wGNkBbWA5UyRFdZWVBEAU0hFgoIH0adPgCY%2BADwCiAVnEAhAILiAnABZxEgOzK1Y%2BQA51M3ROUnJBsbK2WZoyUdkBhcXoAMhlwDFlARnUXZdzE9AGY%2BUgAzDR8QjVdSJQIpGKlhVQQCCIjSUKUfBGEpEJUETFAKVCA .